admin/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
What are these two for? We do not have them in the other html files

Reply:
I don't know, it is automatic with the html template of VScode, do you want it to be removed?

Reply:
Yes, let's keep it as small as possible. We can put it back later when we know what that is for :)

Reply:
This is still not done

Reply:
Ok! I didn't see your answer, thanks for explanation